common_pings.json

```diff
@@ -39,5 +39,9 @@
     {
         "schema_url": "https://raw.githubusercontent.com/mozilla-services/mozilla-pipeline-schemas/master/schemas/telemetry/new-profile/new-profile.4.schema.json",
         "config": "new_profile.yaml"
+    },
+    {
+        "schema_url": "https://raw.githubusercontent.com/mozilla-services/mozilla-pipeline-schemas/master/schemas/telemetry/sync/sync.4.schema.json",
+        "config": "sync.yaml"
```

Comment:
The sync ping is actually not a common ping. MSG adds some fields to `environment` for common pings but since sync doesn't have `environment` it's unaffected. 

Alternatively, instead of handling `sync` as common ping, some logic can be added to support processing generic pings in MSG.